src/components/Section/VestingSection.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+import Markdown from 'decentraland-gatsby/dist/components/Text/Markdown'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+
+import './VestingSection.css'
+
+interface Props {
+  vestingAddress: string
+}
+
+function VestingSection({ vestingAddress }: Props) {
+  const vestingUrl = process.env.GATSBY_VESTING_DASHBOARD_URL
+
+  if (!vestingUrl) {
+    console.error('Vesting Dashboard URL not found')
+    return <></>
+  }
+
+  const t = useFormatMessage()
```

Comment:
Avoid rendering a hook conditionally. Please place this before the vesting Url condition.

Reply:
Thank you! In this particular case I think it would not have been a problem because the environment variable is either there or it is not, but just to be sure I placed the hook before the condition.